lib/api.ts

```
@@ -25,48 +21,59 @@ export interface FetchLogsParams {
+    throw new Error(`Failed to fetch token consumption: ${response.status} ${response.statusText}`)
+  }
+  return response.json()
+}
+
+/**
+ * Fetches current user token information.
+ * @returns User tokens data including available tokens and purchase history
+ */
+export async function fetchUserTokens(): Promise<UserTokensResponse> {
+  // Use the same /api/ prefix pattern as the fetchBotStats function
+  const response = await fetch(`/api/bots/user_tokens`)
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)